main.c

```diff
@@ -347,9 +363,9 @@ int main(void)
             case ST_PHYS_DISCONNECTED:
             case ST_ENUM_SUSPENDED:
             case ST_PHYS_CONNECTED_NOENUM_SUSP:
-                hal_ext_uart(FALSE);
-                poll_hdlc(TRUE);
-                __bis_SR_register(LPM3_bits + GIE);
+                // hal_ext_uart(FALSE);
```

Comment:
What did this do before and why is it now commented out?

Reply:
Keep the serial port always open. It can always accept serial port data and keep TX silent. 
So that's what we talked about before, with batteries, without USB. The radio frequency signal will be bad, mainly because the MSP430 serial port is off, but its TX has entered the suspension state, the battery voltage is not enough, or the battery quality is not good enough, it will continue to input information to the CC1352,  The serial RX interrupt of CC1352 is always in execution state, with only a few time slices to process the wireless signal. @jadonk